Translation/Extractor/File/DefaultPhpFileExtractor.php

```diff
@@ -66,7 +77,7 @@ public function enterNode(\PHPParser_Node $node)
 
         if (!$node instanceof \PHPParser_Node_Expr_MethodCall
             || !is_string($node->name)
-            || ('trans' !== strtolower($node->name) && 'transchoice' !== strtolower($node->name))) {
+            || !in_array(strtolower($node->name), array_keys($this->methodsToExtractFrom))) {
 
```

Comment:
so you only strtolower the $node->name but someone could write the function name as non-lowercase in methodsToExtractFrom. Why even strtolower? should the method name be the same for both? meaning if I do trans in my php code, why would I add Trans to $methodsToExtractFrom ? If there's a reason for that, then shouldn't we be strolowering everything in methodsToExtractFrom? Same question as the change for line 120